src/reset.css

```diff
@@ -0,0 +1,15 @@
+/* #region reset */
+html {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+
+  --section-padding: 16px;
+  --section-width: 166px;
+  --font-color: #060b35;
+}
+/* #endregion */
```

Comment:
remove all comments, you already have separated code into files, so you don't need it